sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsOptions.cs

```diff
@@ -6,7 +6,7 @@ namespace Azure.AI.TextAnalytics
     /// <summary>
     /// Options that allow callers to specify details about how the operation
     /// is run and what information is returned from it by the service.
-    /// <para>For example, whether to include statistics.</para>
```

Comment:
I would prefer to leave as is. the `<para>` helps create a new line which helps with document format.
also not sure we need the `!` either :)